config/kraft/controller.properties

```diff
@@ -31,22 +31,17 @@ controller.quorum.voters=1@localhost:9093
 
 ############################# Socket Server Settings #############################
 
-# The address the socket server listens on. It will get the value returned from
-# java.net.InetAddress.getCanonicalHostName() if not configured.
+# The address the socket server listens on.
+# Note that only the controller listeners are allowed here when `process.roles=controller`.
 #   FORMAT:
 #     listeners = listener_name://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
-listeners=PLAINTEXT://:9093
+listeners=CONTROLLER://:9093
```

Comment:
I wonder if it's helpful to have a note about consistency between this and `controller.quorum.voters`?

Reply:
Good suggestion. Updated.